doc/script_commands.txt

```diff
@@ -2689,6 +2689,82 @@ See 'getequipid' for a full list of valid equipment slots.
 
 ---------------------------------------
 
+*uniqueid_getiteminfo(<"item_unique_id">{,<char_id>})
+
+This function assigns the item information of <item_unique_id> to the variables mentioned down there.
```

Comment:
recommend to add "from the invoking character's inventory if found." for more detailed description